src/Designer/frontend/libs/studio-assistant/src/components/MessageColumn/Messages/AssistantLoadingBubble/AssistantLoadingBubble.module.css

```
@@ -0,0 +1,28 @@
+
+.inlineSpinner {
+  flex-shrink: 0;
+  color: var(--ds-color-text-subtle);
+}
+
+.loadingText {
+  flex: 1 1 auto;
+  animation: assistantPulse 1.5s ease-in-out infinite;
+}
+
+@keyframes assistantPulse {
```

Stylelint (17.12.0)

[error] Expected keyframe name "assistantPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)